app/controllers/main_controller.rb

```diff
@@ -27,7 +27,7 @@ def index
     medium_options = Artwork.is_visible.with_images.all.group(:medium).order(count: :desc).count
 
     @medium_options = medium_options.each_with_index.reduce([]) do |acc, (item, i)|
-      show_by_default = acc.size < 5
+      show_by_default = acc.size < 15
```

Comment:
I don't think we need to bother with this conditional. We should always show all of them.